docs/sources/setup-grafana/configure-grafana/feature-toggles/index.md

```diff
@@ -128,6 +124,7 @@ Experimental features might be changed or removed without prior notice.
 | `logsExploreTableVisualisation`             | A table visualisation for logs in Explore                                                                    |
 | `awsDatasourcesTempCredentials`             | Support temporary security credentials in AWS plugins for Grafana Cloud customers                            |
 | `transformationsRedesign`                   | Enables the transformations redesign                                                                         |
+| `logsActiveLabelsUI`                        | Based on the current query show the active labels in log details                                             |
```

Comment:
```suggestion
| `toggleLabelsInLogsUI`                        | Based on the current query show the active labels in log details                                             |
```

What do you think? As we are adding fully new functionality.

Reply:
Good idea, and then I can add the whole feature behind it.

Reply:
Done.